### ISSUE TITLE
Compactness+Robustness

### ISSUE THREAD

**Issue by VSehwag** [fra31/auto-attack#7]
**Paper**: https://arxiv.org/pdf/2002.10509.pdf

**Venue**: {if applicable, the venue where the paper appeared}

**Dataset and threat model**: CIFAR-10, l-inf, eps=8/255

**Code**: https://github.com/inspire-group/compactness-robustness. A minimal script to evaluate a WRN-28-10 model is available at https://gist.github.com/VSehwag/688632e523df5d2a4c8008f5ee567b1c (only need to download the checkpoint)

**Pre-trained model**: https://www.dropbox.com/sh/56yyfy16elwbnr8/AADmr7bXgFkrNdoHjKWwIFKqa?dl=0 Use the `model_best_dense.pth.tar`

**Log file**: {link to log file of the evaluation}

**Additional data**: yes

**Clean and robust accuracy**: Benign test accuracy = 88.97% , PGD-50 test accuracy (1-restart) = 62.24%, Auto-attack (cheap): 57.88%

**Architecture**: WRN-28-10 (90% connections pruned)

**Description of the model/defense**: Compressed neural networks while simultaneously aching high robustness.

**Comment by fra31** [comment]
Thanks for the submission!

I've run the complete evaluation and added the model to the list.
